src/claimList.js

```diff
@@ -3,49 +3,256 @@ import {
+              you in the past 12 months. If you recently applied, expect an
+              update within the next couple weeks.
+            </div>`}
+      </div>
+      ${oldClaims.length > 0
+        ? html` <h2 style="font-size: 22px; line-height: 32px; margin: 0">
+              Older
+            </h2>
+            <hr
+              style="
+                margin: 0;
+                border: none;
+                border-top: 1px solid #dfe1e2;
+                margin-bottom: 8px;
+              "
+            />
+            <div
+              style="text-transform: uppercase; font-size: 13px; line-height: 16px; margin-bottom: 16px"
+            >
+              started more than a year ago
+            </div>
+            ${oldClaims
+              .map(
+                (claim) => html`
+                  <button
+                    style="	background: none;
+                      color: inherit;
+                      font: inherit;
+                      cursor: pointer;
+                      outline: inherit; 
+                      display: flex;
+                      align-items: center;
+                      border: 0.5px solid #A9AEB1;
+                      padding: 16px 32px;
+                      background-color: #fff;
+                      border-radius: 4px;
+                      gap: 8px;
+                      text-align: left;
+                      margin-bottom: 8px;
+                      width: 100%;
+                      justify-content: space-between"
+                  >
+                    <div>
+                      <b>Claim for ${getClaimTypeContent(claim.type)}</b>,
+                      started ${getFormattedDate(claim.date)}
+                    </div>
+                    <img src="./assets/navigate_next.svg" alt="" />
+                  </button>
+                `
+              )
+              .join("")}`
+        : ""}
+      <div style="margin-bottom: 8px; margin-top: 44px">
+        ${RETURN_TO_TOP_LINK}
+      </div>
+    </div>
+    ${FOOTER_HTML}`;
+  root?.append(newContainer);
 }
 
 function logView(allClaims) {
   const recentClaims = allClaims
-    .map((claim) => claim.claimDate)
+    .map((claim) => claim.date)
     .filter((date) => {
-      const claimDateObj = new Date(date);
+      const dateObj = new Date(date);
       const now = new Date();
       const sixMonthsAgo = new Date().setMonth(now.getMonth() - 6);
-      return claimDateObj > sixMonthsAgo;
+      return dateObj > sixMonthsAgo;
     });
 
   logEvent("[DOL_DABI] Viewed Claim List page", {
-    object_status: allClaims.map((claim) => claim.claimStatus).join(";"),
-    object_type: allClaims.map((claim) => claim.claimType).join(";"),
+    object_status: allClaims.map((claim) => claim.status).join(";"),
+    object_type: allClaims.map((claim) => claim.type).join(";"),
     object_details: recentClaims.length,
     event_label: allClaims.length,
```

Comment:
Dust: I have mixed feelings about how the recent/mixed claims are being passed with these. Ideally I'd rather have it be in some sort for format where each value is more clearly explained (for example: object_details: 'recent claims: x' ), even if it means it requires more parsing downstream. But totally recognize that the audience of folks working with these metrics in analytics/looker is basically you and that it's easier to just have the numbers

Reply:
Ah I see what you're saying. I think it's a fair point, especially as we think about handoff. I also think it's not in the scope of these changes, i'll add it to a ticket to think about making analytics dimensions clearer.